apis/apps/v1alpha1/cluster_types.go

```diff
@@ -92,6 +93,41 @@ type ClusterSpec struct {
 	// customized parameters that is used in different clusterdefinition
 	// +optional
 	Parameters map[string]string `json:"parameters,omitempty"`
+
+	// cluster backup configuration.
+	// +optional
+	Backup *ClusterBackup `json:"backup,omitempty"`
+}
+
+type ClusterBackup struct {
+	// enabled defines whether to enable automated backup.
+	// +kubebuilder:default=false
+	// +optional
+	Enabled bool `json:"enabled,omitempty"`
```

Comment:
pitr enabled?

Reply:
`Enabled` set if enable the automated backup or not. Add another filed `PITREnabled` to set if enable pitr (or log backup)